mlir/test/Dialect/Linalg/vectorize-tensor-extract.mlir

```diff
@@ -113,6 +113,56 @@ module attributes {transform.with_named_sequence} {
   }
 }
 
+// -----
+
+#map1 = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
+func.func @vectorize_scalable_nd_tensor_extract_transfer_read_basic(%arg0: tensor<?x?x?xf32>, %arg2: tensor<?x?x?xf32>) -> tensor<?x?x?xf32> {
```

Comment:
Move to:
* vectorization-scalable.mlir.

It was introduced specifically for scalable vectorisation. This file ("vectorize-tensor-extract.mlir") is for non-masked vectorisation - it uses
* `transform.structured.vectorize_children_and_apply_patterns`

exclusively, which means "no masking". Then, you can also simplify the name as `@vectorize_linalg_index` :)